### ISSUE TITLE
Testing of successfully fine-tuned model does not find any matches and produces 0 TP

### ISSUE THREAD

**Issue by pzalex** [flairNLP/flair#1369]
I am trying to fine-tune an NCBI BlueBERT that is available at [https://github.com/ncbi-nlp/ncbi_bluebert](github repository) for NER tasks. The model comes in format that has to be converted to pytorch format using huggingface transformers-cli available at [https://github.com/huggingface/transformers](github). 

When I fine-tune the BlueBERT with Flair ModelTrainer train() function, I get no errors and no warnings as if the process completes successfully, but during the test phase that follows the training, the model does not select any elements in the test subset, and it shows 0 true positives. I am attaching the [training.log](https://github.com/flairNLP/flair/files/4093721/training.log) file.
The same with annotating text with the newly trained model, I'm getting zero selected elements (0 TP, and 0 FN).

There is no issue with the corpus, the file formatting, or even the code because if I fine-tune the standard Flair BERT with exact same corpus and the same script, I get reasonable results in testing phase 
`PHI        tp: 1540 - fp: 113 - fn: 245 - tn: 1540 - precision: 0.9316 - recall: 0.8627 - accuracy: 0.8114 - f1-score: 0.8958`
, and annotating text with the fine-tuned Flair BERT, I'm getting reasonable results as well consistent with the test phase after training.

Did anyone successfully fine-tune BlueBERT, or ran into the same issue and was able to resolve it? With no errors or warnings during fine-tuning, I do not know where to start investigating.
I'll be happy to share code snippets if needed.
